contracts/FindMarket.cdc

```diff
@@ -48,147 +48,172 @@ pub contract FindMarket {
 		oldProfile.deposit(from: <- vault)
 	}
 
-	pub struct NFTInfo{
+	pub struct NFTInfo {
 		pub let name:String
 		pub let description:String
 		pub let thumbnail:String
 		pub let type: String
```

Comment:
I think we need a Collection View and add that as a subCollection in here really.

Reply:
Yes you are right, I will work on that together with the MarketOption contract today.

Reply:
We need to talk about data details here though. Because I think we need one view that is for details and another one that is for listings/events.

Reply:
id

Reply:
rarity